doc/Tutorials/Introduction.ipynb

```diff
@@ -662,8 +661,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "green05 = levels.Overlay.Green[0.5]\n",
-    "green05 + green05.Channel + green05.Channel.Green.sample(y=0.0)"
+    "green05 = levels.Contours.Green\n",
+    "green05 + chans.Channel.Green + chans.Channel.Green.sample(y=0.0)"
```

Comment:
Doesn't this have implications for backwards compatibility?

Reply:
Guess it has some, it's no longer ``overlaid`` by default. I could have made it be overlaid again but I generally don't want to.